src/Workspaces/Core/Portable/Workspace/WorkspaceFileTextLoader.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+using System.IO;
+using System.Text;
+using System.Threading;
+using Microsoft.CodeAnalysis.Host;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis
+{
+    [DebuggerDisplay("{GetDebuggerDisplay(), nq}")]
+    internal class WorkspaceFileTextLoader : FileTextLoader
```

Comment:
I admit I don't understand this class name or what it's really representing here. Doc it?